src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -230,6 +231,15 @@ Copyright (c) .NET Foundation. All rights reserved.
       <ExpectedPlatformPackages Include="@(PackageReference)" Condition="'%(Identity)' == 'Microsoft.AspNetCore.All'" />
     </ItemGroup>
 
+    <CheckForTargetInAssetsFile
```

Comment:
I think this will be a perf regression, forcing the assets file to be read from disk on every incremental build. I think it used be like this in fact, and I folded it into ResolvePackageAssets for this reason.